mne/source_estimate.py

```diff
@@ -3222,7 +3222,6 @@ def stc_near_sensors(evoked, trans, subject, distance=0.01, mode='sum',
     For surface projections, this function projects the ECoG sensors to
     the pial surface (if ``project``), then the activation at each pial
     surface vertex is given by the mode:
-
```

Comment:
In other words, this is probably going to break things

Reply:
Fixed.